### PR TITLE
#71 Updated description message for LocateFile TU

### PR DESCRIPTION
**Pull request by amriteya**
Description message for cases where the transformation context is not available or the relative file had not been set was automatically set `root folder`.
Changed the description to more generic message to handle cases for Locate file which can only be determined at runtime.

### CODE REVIEW

File: butterfly-utilities/src/main/java/com/paypal/butterfly/utilities/file/LocateFile.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
  */
 public class LocateFile extends TransformationUtility<LocateFile> {
 
+    private static final String DESCRIPTION_RELATIVE_MISSING = "Locate file cannot be determined";
```
Comment (fabiocarvalho777) [inline butterfly-utilities/src/main/java/com/paypal/butterfly/utilities/file/LocateFile.java]: @amriteya please rephrase this to `Locate specified file`. This sounds more gentle I believe.

File: butterfly-utilities/src/test/java/com/paypal/butterfly/utilities/file/LocateFileTest.java [hunk 2]
```diff
@@ -25,7 +25,7 @@ public void rootLocateFileTest() throws IOException {
         File rootFile = (File) executionResult.getValue();
         Assert.assertEquals(rootFile.getAbsolutePath(), transformedAppFolder.getAbsolutePath());
         Assert.assertEquals(locateFile.getParentLevel(), 0);
-        Assert.assertEquals(locateFile.getDescription(), "Locate file root folder");
+        Assert.assertEquals(locateFile.getDescription(), "Locate file cannot be determined");
```
Comment (fabiocarvalho777) [inline butterfly-utilities/src/test/java/com/paypal/butterfly/utilities/file/LocateFileTest.java]: Change this according to comment above.

**Comment by amriteya** [comment]
@fabiocarvalho777 Addressed the review comments.
